tests/LoginPage.spec.js

```diff
@@ -0,0 +1,79 @@
+import { test, expect } from "@playwright/test";
+import credentials from "../Fixtures/credentials.json";
+import generalData from "../Fixtures/generalData.json";
+import { LoginPage } from "../PageObjects/UI/LoginPage.js";
+import { RegisterPage } from "../PageObjects/UI/RegisterPage.js";
+import { GeneralMethods } from "../Fixtures/generalMethods.js";
+const generalMethods = new GeneralMethods();
+
+test.describe("Tests for Login page", () => {
+  test("Checkout Login page elements", async ({ page }) => {
```

Comment:
never import class, you need page fixtures here... explore how they work in playwright so you dont lose time and lines to initiate class every time you use it.